yarn-project/Earthfile

```diff
@@ -58,8 +62,14 @@ aztec-prod-slim:
 
 aztec:
     FROM +aztec-prod-slim
+    RUN apt update && apt install -y curl && rm -rf /var/lib/apt/lists/* && apt-get clean
+    LET port=8080
+    ENV PORT=$port
+    HEALTHCHECK --interval=10s --timeout=10s --retries=6 \
+        CMD curl -fsSd '{\"jsonrpc\":\"2.0\",\"method\":\"pxe_getNodeInfo\",\"id\":1}' http://127.0.0.1:$port
```

Comment:
Thoughts on using docker healthchecks instead of netcat? We'd be able to do it in one place (the `aztec` image) and then anyone running the image would benefit from using a fully-bootstrapped sandbox. This relies on the sandbox only binding to the port after its finished setting up (which is the case currently).